apps/application-system/api/src/app/modules/payment/payment.service.ts

```diff
@@ -44,18 +67,21 @@ export class PaymentService {
         payment.application_id) as string) +
       this.paymentConfig.callbackAdditionUrl +
       payment.id
+    const paymentDefinition = JSON.parse(payment.definition as string)
     const charge: Charge = {
       // TODO: this needs to be unique, but can only handle 22 or 23 chars
       // should probably be an id or token from the DB charge once implemented
       chargeItemSubject: payment.id.substring(0, 22),
       systemID: 'ISL',
-      performingOrgID: '6509142520',
+      // The OR values can be removed later when the system will be more robust.
+      performingOrgID:
+        paymentDefinition?.performingOrganizationID || '6509142520',
       payeeNationalID: user.nationalId,
-      chargeType: 'AY1',
+      chargeType: paymentDefinition?.chargeType || 'AY1',
       performerNationalID: user.nationalId,
       charges: [
         {
-          chargeItemCode: 'AY110',
+          chargeItemCode: paymentDefinition?.chargeItemCode || 'AY110',
```

Comment:
Great observation, again it is up for debate - this is only temporary hard coding, which I would hope to be solved within next two weeks.
Right now the payment is hard coded for Driving Licence's, but it will become generic soon - which means no hard coding.
The debate lies with putting work & crowding env variables temporary or letting it be and perhaps sticking to comments?